packages/ui-components/Common/AvatarGroup/Avatar/index.stories.tsx

```diff
@@ -1,15 +1,14 @@
 import type { Meta as MetaObj, StoryObj } from '@storybook/react';
 
-import Avatar from '@/components/Common/AvatarGroup/Avatar';
-import { getGitHubAvatarUrl } from '@/util/gitHubUtils';
+import Avatar from '@node-core/ui-components/Common/AvatarGroup/Avatar';
 
 type Story = StoryObj<typeof Avatar>;
 type Meta = MetaObj<typeof Avatar>;
 
 export const Default: Story = {
   args: {
-    image: getGitHubAvatarUrl('ovflowd'),
-    nickname: 'ovflowd',
+    image: 'https://avatars.githubusercontent.com/avivkeller',
+    nickname: 'avivkeller',
```

Comment:
:P

Reply:
Instead of yours or mine, use the username "ghost"

Reply:
Sure thing! Sorry about taking your thunder, I originally went off user IDs, and had mine saved, which made it easier.

(I later realized usernames also work for avatar URLs, so that wasn't even really needed)